src/data/nav.yml

```diff
@@ -146,6 +146,22 @@
           url: '/build-apps/ab-test/subscribe'
     - title: See all open source apps
       url: https://opensource.newrelic.com/nerdpacks/
+
+- title: Instant observability
+  icon: zap
+  url: '/instant-observability'
```

Comment:
I suggest we follow the existing pattern of action oriented Navigation and call this Build a Quickstart. By call it instant observability it will confuse users as we will have a top level navigation element of the same name. That one takes a user to the I/O expierence, where this one will take a user to a different page.


I found that  bit confusing, so I'd suggest we bring up the using of Build a quickstart and make that the primary nav item.

Reply:
The navigation is action oriented but also scoped at topic level. Build a quickstart is a very narrow focus and as planned, we'll be adding more labs under the "Instant Observability" topic. 

I'm going to name the top level directory as "Contribute to Instant Observability" which aligns with both the scope and the action oriented navigation style.

Reply:
hmm that seems like a really long nav item. It ends up wrapping in the navigation component. 

You may want to ask for an opinion on this from @bradleycamacho also I'd like to see what @jaesius thinks.

  I personal think we can avoid the term I/O at this level in the navigation as it's just a very long set of words and seems to be unclear as to why I'd click on that, as the repo itself where you build things is all about quickstarts, I/O is really a concept.



 

<img width="315" alt="Screen Shot 2021-09-28 at 1 42 10 PM" src="https://user-images.githubusercontent.com/1395158/135155025-2da2059e-fe7f-4a1e-9240-34ab539b0aef.png">

Reply:
I agree with John. I think `Contribute to Instant Observability` is too long for the main navigation and doesn't align with any of the other New Relic properties. 

Also, making the connection between Quickstarts and Instant Observability isn't needed on this level of navigation. 

I naturally lean towards something like "Build A Quickstart."